python/modules/IcePy/Communicator.cpp

```diff
@@ -70,112 +71,22 @@ communicatorNew(PyTypeObject* type, PyObject* /*args*/, PyObject* /*kwds*/)
     return self;
 }
 
+// Only called by Ice.Initialize
+// IcePy.Communicator(args: [str]|None, initData: Ice.InitializationData|None, configFile: str|None)
+
 extern "C" int
 communicatorInit(CommunicatorObject* self, PyObject* args, PyObject* /*kwds*/)
 {
-    //
```

Comment:
I move the validation of the parameters to Ice.initialize, which is the only caller of IcePy.Communicator.